aws_marketplace/using_model_packages/preprocessing-audio-data-using-a-machine-learning-model/preprocessing-audio-data-using-a-machine-learning-model.ipynb

```diff
@@ -0,0 +1,868 @@
+{
```

Comment:
<p>We have S3 download helpers in the Python SDK. Is this doing more than just downloading?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3061/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
<p>The output of the batch transform job is actually a compressed zip file. However, the extension does not reflect that. Hence, I add the <code>.zip</code> extension while downloading each file.</p>